addon/templates/components/polaris-data-table/navigation.hbs

```diff
@@ -0,0 +1,19 @@
+{{polaris-button
+  plain=true
+  icon="chevron-left"
+  disabled=currentColumn.isScrolledFarthestLeft
+  accessibilityLabel="Scroll table left one column"
+  onClick=(action navigateTableLeft)
+}}
+
+{{#each columnVisibilityData as |column|}}
+  <div class="Polaris-DataTable__Pip {{if column.isVisible "Polaris-DataTable__Pip--visible"}}"></div>
```

Comment:
Maybe single-quote the inner string for slightly easier readability? No biggie if we prefer double quotes even for nested strings in templates, but just a thought.

Reply:
Fair point, never single-quoted in template files before. I'm OK with this as is, but if both you and @vladucu would prefer it changed I'm happy to change it, no biggie as you say 😉

Reply:
I've usually used single-quotes in cases like this just to avoid confusion and be easier to read